### ISSUE TITLE
hard coded strings in commentsProxy.cfc 

### ISSUE THREAD

**Issue by j-herden** [blueriver/MuraCMS#2687]
There are hard coded strings which can not get translated:

display_objects/comments/ajax/commentsProxy.cfc 

<em>(in reply to: <a href="##" class="mura-in-reply-to" data-parentid="#comment.getParentID()#">#comment.getParent().getName()#</a>)</em>

and 

<div class="mura-comment-more-comments-container #renderer.commentMoreCommentsContainer#"><a id="mura-more-comments" class="#renderer.commentMoreCommentsDownClass#" href="##" data-pageno="#it.getPageIndex()+1#" data-siteid="#HTMLEditFormat($.event('siteID'))#">More Comments</a></div>

**Comment by cfmitrah** [comment]
Yes, I can see the hard coded strings for the texts you mentioned. I've added the variable for these strings in regarding languages files.
For the admin directory I've added the language in the "requirements\mura\resourceBundle\resources\" directory.
For the default directory I've added in the "/default/includes/resourceBundles" directory.
Also, I've noticed that these strings not only used in this (comments\ajax\commentsProxy.cfc) file. But also the in other couple of files too. I've replaced the variable names in those files too.

filenames:
	default\includes\display_objects\comments\ajax\commentsProxy.cfc
	admin\core\views\ccomments\loadcommentspage.cfm

I've created the PR for this issue.

Pull Request for this ticket : #2798

**Comment by cfmitrah** [comment]
I've created the PR for the 7.1 also. Please check.

Pull Request for this ticket : #2812
